### PR TITLE
[air] update rllib example to use Tuner API.

### PR DESCRIPTION
**Pull request by xwjiang2010**
## Why are these changes needed?
update rllib example to use Tuner API.

## Checks

- [X] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [x] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: rllib/examples/vizdoom_with_attention_net.py [hunk 1]
```diff
@@ -73,14 +73,17 @@
...
 
-    results = tune.run(
+    results = tune.Tuner(
         args.run,
-        config=config,
-        stop=stop,
-        verbose=2,
-        checkpoint_freq=5,
-        checkpoint_at_end=True,
-        restore=args.from_checkpoint,
```
Comment (krfricke) [inline rllib/examples/vizdoom_with_attention_net.py:83]: I guess we don't support the restore argument without the RLTrainer - should we remove `args.from_checkpoint` or maybe pass the tuner kwargs?

**Review by krfricke — APPROVED** [review]
Looks great, one minor thing, your call what to do with it

File: rllib/examples/vizdoom_with_attention_net.py (hunk 1 above)
Comment (xwjiang2010) [reply rllib/examples/vizdoom_with_attention_net.py:83]: Yes, that's correct.
Prefer to just remove it. (I think I don't have lingering `args.from_checkpoint`? Or do you see any?)

Comment (krfricke) [reply rllib/examples/vizdoom_with_attention_net.py:83]: Ah I think it's in line 15

**Comment by xwjiang2010** [comment]
Ah right, removed!! Thanks :)

**Pull request merged**